### PR TITLE
Add venv to install instructions page

### PR DESCRIPTION
**Pull request by greg-mckeon**
From a convo with @jtcohen6, we'd really like all of our core users to be using a virtual environment of some kind.  This makes that recommendation more explicit by moving it from a collapsed tab to the install page itself.

### CODE REVIEW

File: website/docs/docs/core/pip-install.md [hunk 1]
```diff
@@ -5,14 +5,36 @@ description: "You can use pip to install dbt Core and adapter plugins from the c
...
-You can install dbt Core and plugins using `pip` because they are Python modules distributed on [PyPI](https://pypi.org/project/dbt/). We recommend using virtual environments when installing with `pip`.
-
+You can install dbt Core and plugins using `pip` because they are Python modules distributed on [PyPI](https://pypi.org/project/dbt/).
 
 <FAQ path="Core/install-pip-os-prereqs" />
 <FAQ path="Core/install-python-compatibility" />
-<FAQ path="Core/install-pip-best-practices" />
 
+### Using virtual environments
+We recommend using virtual environments to namespace pip modules. To create a new venv:
```
Comment (runleonarun) [inline website/docs/docs/core/pip-install.md]:
```suggestion
We recommend using virtual environments (venv) to namespace pip modules. 

1. Create a new venv:
```

File: website/docs/docs/core/pip-install.md [hunk 2]
```diff
@@ -5,14 +5,36 @@ description: "You can use pip to install dbt Core and adapter plugins from the c
...
+
+```shell
+python3 -m venv dbt-env				# create the environment
+```
+
+If you install dbt in a virtual environment, you need to reactivate that same virtual environment each time you create a shell window or session.
```
Comment (runleonarun) [inline website/docs/docs/core/pip-install.md]:
```suggestion
2. Reactivate the dbt virtual environment with every new session. When you install dbt in a virtual environment, you need to reactivate that same virtual environment each time you open a new shell window or session.
```

File: website/docs/docs/core/pip-install.md [hunk 3]
```diff
@@ -5,14 +5,36 @@ description: "You can use pip to install dbt Core and adapter plugins from the c
...
+
+```shell
+source dbt-env/bin/activate			# activate the environment for Mac and Linux OR
+dbt-env\Scripts\activate			# activate the environment for Windows
+```
+
+:::tip
+You can create an alias for the source command in your $HOME/.bashrc, $HOME/.zshrc, or whichever rc file your shell draws from.
```
Comment (runleonarun) [inline website/docs/docs/core/pip-install.md]: I suggest calling it a "config" which feels more universally understood. I know rc is technically more than a config, but hopefully config works here.
```suggestion
You can create an alias for the source command in your $HOME/.bashrc, $HOME/.zshrc, or whichever config file your shell draws from. 
```

File: website/docs/docs/core/pip-install.md [hunk 4]
```diff
@@ -5,14 +5,36 @@ description: "You can use pip to install dbt Core and adapter plugins from the c
...
+```
+
+If you install dbt in a virtual environment, you need to reactivate that same virtual environment each time you create a shell window or session.
+
+```shell
+source dbt-env/bin/activate			# activate the environment for Mac and Linux OR
+dbt-env\Scripts\activate			# activate the environment for Windows
+```
+
+:::tip
```
Comment (runleonarun) [inline website/docs/docs/core/pip-install.md]: Let's use a heading so we can link to it above. And so others can link directly to it if necessary..
```suggestion
#### Create an alias
```

File: website/docs/docs/core/pip-install.md [hunk 5]
```diff
@@ -5,14 +5,36 @@ description: "You can use pip to install dbt Core and adapter plugins from the c
...
+You can create an alias for the source command in your $HOME/.bashrc, $HOME/.zshrc, or whichever rc file your shell draws from. 
+
+For example, add the following to your rc file, replacing <PATH_TO_VIRTUAL_ENV_CONFIG> with the path to your virtual environment configuration.
+
+```shell
+alias env_dbt='source <PATH_TO_VIRTUAL_ENV_CONFIG>/bin/activate'
+```
+:::
```
Comment (runleonarun) [inline website/docs/docs/core/pip-install.md]: Need to remove if we're using a heading.
```suggestion
```

**Review by runleonarun — CHANGES_REQUESTED** [review]
@greg-mckeon Just a few minor requests!

File: website/docs/docs/core/pip-install.md (hunk 2 above)
Comment (greg-mckeon) [reply website/docs/docs/core/pip-install.md]: Changed this up a bit to be shorter.

**Comment by greg-mckeon** [comment]
Updated, feel free to submit if it looks good!

File: website/docs/docs/core/pip-install.md [hunk 6]
```diff
@@ -5,14 +5,37 @@ description: "You can use pip to install dbt Core and adapter plugins from the c
...
+
+2. Activate that same virtual environment each time you create a shell window or session:
+
+```shell
+source dbt-env/bin/activate			# activate the environment for Mac and Linux OR
+dbt-env\Scripts\activate			# activate the environment for Windows
+```
+
+#### Create an alias
+You can create an alias for the source command in your $HOME/.bashrc, $HOME/.zshrc, or whichever config file your shell draws from.
```
Comment (runleonarun) [inline website/docs/docs/core/pip-install.md]:
```suggestion
To activate your dbt environment with every new shell window or session, you can create an alias for the source command in your $HOME/.bashrc, $HOME/.zshrc, or whichever config file your shell draws from. 
```

**Review by runleonarun — APPROVED** [review]
🚢 it

**Pull request merged**